package com.uhm.uhmcs.bean;



import java.util.ArrayList;

/**
 * 商品分类列表
 */
public class CategoryListBean {


    private int code;

    private String msg;

    private String time;


    private ArrayList<CategoryListModel> data;

    public ArrayList<CategoryListModel> getData() {
        return data;
    }

    public void setData(ArrayList<CategoryListModel> data) {
        this.data = data;
    }
    public int getCode() {
        return code;
    }

    public void setCode(int code) {
        this.code = code;
    }

    public String getMsg() {
        return msg;
    }

    public void setMsg(String msg) {
        this.msg = msg;
    }

    public String getTime() {
        return time;
    }

    public void setTime(String time) {
        this.time = time;
    }



    public static class CategoryListModel{


        private String id="";
        private String shop_id;
        private String category_id="";
        private String name;
        private String image;
        private int drawableId;
        private String createtime;




        public int getDrawableId() {
            return drawableId;
        }

        public void setDrawableId(int drawableId) {
            this.drawableId = drawableId;
        }

        public String getId() {
            return id;
        }

        public void setId(String id) {
            this.id = id;
        }

        public String getShop_id() {
            return shop_id;
        }

        public void setShop_id(String shop_id) {
            this.shop_id = shop_id;
        }

        public String getCategory_id() {
            return category_id;
        }

        public void setCategory_id(String category_id) {
            this.category_id = category_id;
        }

        public String getName() {
            return name;
        }

        public void setName(String name) {
            this.name = name;
        }

        public String getImage() {
            return image;
        }

        public void setImage(String image) {
            this.image = image;
        }

        public String getCreatetime() {
            return createtime;
        }

        public void setCreatetime(String createtime) {
            this.createtime = createtime;
        }
    }
}
